wiki/en/misc/1-index.html

```diff
@@ -1,23 +1,23 @@
 ---
-title: "Jamulus ‒ Play music online. With friends. For free."
+title: "Jamulus — Play music online with friends."
 lang: "en"
 permalink: /
 layout: mainhomepage
 ldjAppCategory: "Music"
-ldjSlogan: "Play music online. With friends. For free."
+ldjSlogan: "Play music online with friends."
 ldjCreator: "Volker Fischer and contributors"
-metadescription: "Jamulus is open source software which enables musicians to perform music, rehearse or just jam, all in real time over the Internet."
+metadescription: "Copylefted libre (<a href="https://wikiless.org/wiki/GNU_General_Public_License?lang=simple" target="_blank" rel="noreferrer">GPLv3+</a>) software for musicians to perform, rehearse or just jam together — all in real-time over the Internet."
```

Comment:
```suggestion
metadescription: "Copylefted libre (GPLv2+) software for musicians to perform, rehearse or just jam together — live over the Internet."
```
@gilgongo The app is GPLv2-only in https://hosted.weblate.org/projects/jamulus/#information which is the issue to solve in https://github.com/jamulussoftware/jamulus/issues/2830
(I am happy to re-license my strings, but strictly speaking contributors must be asked.)